broker/src/util.erl

```diff
@@ -58,6 +58,16 @@ interpolate(Text, Fun, Output) ->
       end
   end.
 
+interpolate_js(Text, JsContext) ->
```

Comment:
I don't like this name very much. It's not descriptive enough IMHO. Maybe `interpolate_using_js`?